examples/rust/digital_signature/src/main.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Risc0, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+fn main() {
+    todo!()
```

Comment:
Can totally leave this as a TODO for this commit, but I think a tiny command line util that supported 'sign' and 'verify' and that took like string inputs and hashed them (on the 'host') might be cool in some future commit.

Reply:
Updated in the latest commit. One thing I noticed: there doesn't seem to be an API for serializing/deserializing `r0vm_host::Proof`. So while I've implemented a command-line tool for signing, it does not currently provide any way to save the receipt or verify it later :/